prometheus_api_client/prometheus_connect.py

```diff
@@ -35,16 +38,27 @@ class PrometheusConnect:
 
     def __init__(
             self, url: str = "http://127.0.0.1:9090", headers: dict = None,
-            disable_ssl: bool = False
+            disable_ssl: bool = False, retry: Retry = None
```

Comment:
Could you please add a small description [here](https://github.com/AICoE/prometheus-api-client-python/blob/master/prometheus_api_client/prometheus_connect.py#L32) for this new parameter?

Reply:
done